styles.css

```diff
@@ -0,0 +1,124 @@
+body {
+  font-family: "poppins";
+  background-color: aliceblue;
+}
+.card {
+  max-width: 90%;
+  margin-left: 5%;
+  background-color: white;
+  box-shadow: 10px 20px 10px lightgray, -10px 0px 10px lightgray;
+  padding: 10px;
+}
+h1 {
+  text-align: center;
+  font-size: 25px;
+  font-weight: lighter;
+  opacity: 0.6;
+  letter-spacing: -1px;
+  margin-top: 50px;
+}
+h2 {
+  text-align: center;
+  font-size: 25px;
+  font-weight: bolder;
+  margin-top: -20px;
+  opacity: 0.9;
+}
+header > p {
+  text-align: center;
+  font-size: 15px;
+  opacity: 0.8;
+  max-width: 40%;
+  padding-left: 30%;
+  margin-top: -20px;
+}
+.content {
+  display: grid;
+  grid-template-columns: repeat(3, 280px);
+  grid-template-rows: repeat(4, 90px);
+  grid-gap: 20px;
+  max-width: fit-content;
+  margin: 0 auto 5%;
+}
+.supervisor {
+  grid-area: 2/1/4/2;
+  border-top: 3px solid teal;
+}
+.builder {
+  grid-area: 1/2/3/3;
+  border-top: 3px solid red;
+}
+.karma {
+  grid-area: 3/2/5/3;
+  border-top: 3px solid yellow;
+}
+.calculator {
+  grid-area: 2/3/4/4;
+  border-top: 3px solid rgb(0, 81, 128);
+}
+.section {
+  box-shadow: 5px 10px 10px lightgray, -5px 0px 10px lightgray;
+  padding: 0px 20px;
+}
+h3 {
+  font-size: 18px;
+  font-weight: 800;
+  font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
+}
+section > p {
+  opacity: 0.8;
+  font-size: 12px;
+  letter-spacing: 0.5px;
+}
+img {
+  padding: 30px 0px 0px 190px;
+  width: 50px;
+}
+
+@media screen and (min-width: 370px) and (max-width: 576px) {

```

Comment:
Because of this line, all screens below 370px width will render the desktop styles :)

Reply:
Just gonn take out the min-width then..